AWS/legos/aws_list_expiring_acm_certificates/aws_list_expiring_acm_certificates.json

```diff
@@ -9,5 +9,8 @@
     "action_supports_iteration": true,

```

Comment:
Also for checks , the output type will always be a LIST of dicts, so basically ACTION_OUTPUT_TYPE_LIST